app/views/posts/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="posts container">
+
+  <div class="user_item row">
+    <div class="u_img">
+      <img src=<%= @user.photo %> alt="User Image">
+    </div>
+
+    <div class="user_cont">
+      <a href=<%= user_url(@user.id) %>>
+        <div class="user_desc col">
+          <h2 class="user_name"><%= @user.name %></h2>
+          <p class="user_post_n">Number of posts: <%= @user.posts_count %></p>  
+        </div>
+      </a>
+    </div> 
+  </div>
+
+  <ul class="post_list post_list_show col">
+    <% @set_posts.each do |post| %>
+      <li class="post_item_desc">
+
+        <a href=<%= user_post_url(user_id: @user.id, id: post.id) %>>
+          <div class="post_item">
+            <h4 class="post_title"><%= post.title %></h4>
+            <p class="post_text"><%= post.text %></p>
+            <p class="likes">
+              Comments: <%= post.comments_count %>, 
+              Likes: <%= post.likes_count %>
```

Comment:
- [ ] Please use correct table column name for now you using `comments_count` and `likes_count` but in database table column name is `comment_counter` and `likes_counter` so kindly fix because its give an error check :slightly_smiling_face: